templates/new/src/config/.entry.js

```diff
@@ -28,6 +28,10 @@ export default class Entry extends Component {
 }
 
 Entry.start = function (getRoutes, getStore, match=originalMatch, history=browserHistory) {
+  // Allow developers to include code that will be executed before the app is
+  // set up in the browser.
+  require("./init.browser");
```

Comment:
This is another power tool that lets developers execute code as soon as the app is mounted. Needed for Aphrodite:
![image](https://cloud.githubusercontent.com/assets/140000/20199693/4f6d9a62-a761-11e6-81ca-c97c59ff2828.png)